plugins/pay.c

```diff
@@ -1733,7 +1733,11 @@ static void add_new_entry(struct json_stream *ret,
 			  const struct pay_mpp *pm)
 {
 	json_object_start(ret, NULL);
-	json_add_string(ret, "bolt11", pm->b11);
+	if(pm->b11){
+		json_add_string(ret, "bolt11", pm->b11);
+	}else{
```

Comment:
Nit: whitespace arount the parentheses: 

```suggestion
	if (pm->b11) {
		json_add_string(ret, "bolt11", pm->b11);
	} else {
```